pkgs/tools/archivers/star/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "star-${version}";
+  version = "1.5.3";
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/s-tar/files/${name}.tar.gz";
```

Comment:
Should use `mirror://sourceforge`. Something like
```
url = "mirror://sourceforge/s-tar/${name}.tar.gz";
```
might work.

There also seems to be a tar.bz2 download file as well so can use that. Saves a few bytes :-)